source/globals.d.ts

```diff
@@ -70,11 +72,10 @@ interface HTMLFormControlsCollection {
 }
 
 declare module '*.svg' {
-	export default (): SVGElement => SVGElement;
+	export default (): JSX.Element => JSX.Element;
```

Comment:
Sad point: you can't use regular DOM-returning functions as `<Fun>`, they must return `JSX.Element`, which only exists the DOM is already generated with JSX (or its overridden this way)